packages/bpk-component-card-list/src/BpkCardListRowRail/BpkCardListCarousel.module.scss

```diff
@@ -52,6 +58,31 @@
       overflow: visible;
       box-sizing: border-box;
       scroll-snap-align: start;
+
+      @include breakpoints.bpk-breakpoint-mobile {
+        &:first-child {
+          padding-left: tokens.bpk-spacing-sm();
```

Comment:
On mobile, the first card should be as close as possible to monetization. Considering the shadow, it cannot be set to 0 completely.
<img width="1382" height="667" alt="image" src="https://github.com/user-attachments/assets/4abd7050-a7e9-4f49-8142-7994d96ddc13" />